### DISCUSSION TITLE
Problem connecting to Security Onion Web Interface

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#9476] [Unsupported Versions]
Help please. I tried to access my security onion web management interface but was unsuccessful. I used the IP address from my ubuntu box. I did select DHCP for the management interface which is what I suspect is causing this problem. Is there a means to revert that setting and make it static instead? You know, without having to do a re-installation just to fix that one issue. Thanks.

**Comment by alexandruhera** [comment]
You can run `sudo SecurityOnion/setup/so-setup iso` and chose to reconfigure the network instead of reinstalling the whole thing.
Additionally, add your Ubuntu box to allowed IPs that can connect to the analyst console (TCP 443).

**Reply by ghost** [reply]
Ok so I tried it and went through the remainder of the configuration process. Unfortunately it gives me the output attached. Since the machine is not even up can I still access the folder specified? And how can I resolve it?
![SO status](https://user-images.githubusercontent.com/113719151/209723581-189c1883-0ea4-4b1b-bd43-9d81c11dc511.png)

**Reply by dougburks** [reply]
You should be able to re-run Setup via SSH or the console. Make sure that your network configuration is correct and that you can connect to the Internet. If you don't have Internet access but are using our Security Onion ISO image, then you can choose the Airgap option:
https://docs.securityonion.net/en/2.3/airgap.html
